### DISCUSSION TITLE
Welcome to fixitwizard Discussions!

### DISCUSSION THREAD

**Discussion by MoHiTKuKrEjA11** [MoHiTKuKrEjA11/fixitwizard#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MoHiTKuKrEjA11** [comment]
Why is the footer not showing in the signup/signin page

**Reply by MKD2002** [reply]
I think it is because of the z-index that we have applied
